crates/uv-cli/src/lib.rs

```diff
@@ -1833,6 +1833,14 @@ pub struct RunArgs {
     #[arg(long)]
     pub with: Vec<String>,
 
+    /// Run with all packages listed in the given `requirements.txt` files.
+    ///
+    /// Using `pyproject.toml`, `setup.py`, or `setup.cfg` files is not allowed.
+    ///
+    /// If `-` is provided, then requirements will be read from stdin.
```

Comment:
This seems dubious here, is it easy to disable?

Reply:
I don't think so...

Reply:
Checking

Reply:
Fixed